src/locale/locales/fr/messages.po

```diff
@@ -1,4 +1,4 @@
-msgid ""
+.msgid ""
```

Comment:
```suggestion
msgid ""
```

(Here is a suggestion you can easily apply using GitHub's UI to get rid of that pesky `.` at the beginning of the file, and which prevents me from actually saying "LGTM" 😅)